lib/process_win32.cpp

```diff
@@ -3,6 +3,32 @@
 
 #include "rtl_exec.h"
 
+struct ThreadInfo {
+    HANDLE pHandle;
+    std::vector<unsigned char> *buffer;
+};
+
+static DWORD WINAPI reader_thread(LPVOID lpParam)
+{
+    ThreadInfo *ti = reinterpret_cast<ThreadInfo*>(lpParam);
+
+    while (ti->pHandle != INVALID_HANDLE_VALUE) {
```

Comment:
Same about no need to check for invalid handle.

Reply:
Fixed.